tests/Load/Dockerfile

```
@@ -1,18 +1,11 @@
-FROM golang:1.23-alpine AS builder
+FROM golang:1.24-alpine AS builder
 
-WORKDIR /app
+RUN apk add --no-cache git
 
-ENV CGO_ENABLED=0
-RUN go install go.k6.io/xk6/cmd/xk6@v0.11.0
+RUN go install go.k6.io/xk6/cmd/xk6@latest && \
```

Hadolint (2.14.0)

[warning] Pin versions in go. Instead of `go install <package>` use `go install <package>@<version>`

(DL3062)